routes/index.js

```diff
@@ -10,4 +11,6 @@ router.get('/', get)
 router.use('/ping', pingRouter)
 router.use('/categories', categoriesRouter)
 
+router.use('/news/:id', newsRouter)
```

Comment:
```suggestion
router.use('/news', newsRouter)
```
estoy casi seguro que esto va en la ruta de news